app/tests/manual/router.ts

```diff
@@ -253,7 +253,7 @@ angular
                                         label: 'All States',
                                         value: ''
                                     },
-                                    { name: 'Alabama', label: 'Alabama', value: 'AL' },
+                                    { name: 'Alabama', label: 'Alabama Long Multi-Word Title', value: 'AL' },
```

Comment:
are you using "Long Multi-Word Title" in some way?

Reply:
This & the other one you mentioned are just test values so if you're looking at the component in the share components test site, you can see the non-wrapping in action.

Reply:
ok very nice